src/easymotion/easymotion.ts

```diff
@@ -294,8 +324,6 @@ export class EasyMotion {
       // Get keys after the depth we're at
       var keystroke = marker.name.substr(this.accumulation.length);
 
-      let fontColor = keystroke.length > 1 ? "orange" : "red";
```

Comment:
I am disappointed to take this out but am not sure how else I could accomplish it